mappings/net/minecraft/world/gen/feature/GlowstoneBlobFeature.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_115 net/minecraft/world/gen/feature/GlowstoneBlobFeature
```

Comment:
Related to the clay patch naming change, I would suggest renaming this to "GlowstoneClusterFeature" due to the crystals forming a cluster around a somewhat central point. This shape becomes more obvious when increasing the size of these clusters.

Reply:
I like this, it kinda better describes what the formation is as its not a solid "blob" formation but a crystal one